CodingStandards.txt

```diff
@@ -91,8 +88,6 @@ All other entities' first alpha is lower case.
 4. Variable prefixes:
 
 a. Leading underscore "_" to parameter names (both normal and template).
-- Exception: "o_parameterName" when it is used exclusively for output. See 6(f).
```

Comment:
Why not flag this extremely rare pattern in the parameter name, like `outputParameterName` rather than having an invented special naming convention?

Or indeed, why bother having a standard at all for something which is extremely rare?  Such things should just have extensive comments, so that they are really noticeable.   Not just noticeable because they use a weird variable prefix.